### DISCUSSION TITLE
How much am I at risk using this

### DISCUSSION THREAD

**Discussion by epic-git** [victornpb/undiscord#199] [1. Questions & Answers] [answered]
will i get detected if i use it?

**Comment by ghost** [comment, marked as answer]
Your questions will more likely be answered when you post them in the [Issues tab](https://github.com/victornpb/deleteDiscordMessages/issues) of [this repository](https://github.com/victornpb/deleteDiscordMessages/).
Refer to https://github.com/victornpb/deleteDiscordMessages/issues/193 for more information about your question.
